MeasureAuthoringTool/mat-drupal
# Drupal custom blocks

## Description
Creates and themes Hero block. Creates and themes footer blocks (CMS Branding, Resource Navs). Hero block and CMS branding block are provided by a module. Resource Nav blocks are system menu blocks.

## To Test
1. Import drupal config `docker-compose exec drupal drush cim -y`
2. Compile theme `cd html/themes/custom/emat && npm start`
3. Clear cache `docker-compose exec drupal drush cr`
4. Add some links to the new menus:
  - `/admin/structure/menu/manage/cms-hhs-websites`
  - `/admin/structure/menu/manage/helpful-links`
5. Visit the homepage, see that it looks something like the attached screenshot

![Screenshot_2020-02-24 Welcome to Measure Authoring Tool Measure Authoring Tool](https://user-images.githubusercontent.com/1919263/75188540-e79b5a00-5711-11ea-96a0-78a05aaa1d26.png)
